docker-compose.yml

```diff
@@ -0,0 +1,47 @@
+version: "3.4"
+services:
+  database:
+    # When this is updated, we'll have to handle creating the postgis extension
+    # somewhere in the test workflow. That's handled automatically on 0.1.0 but
+    # not on later versions
+    image: quay.io/azavea/postgis:0.1.0
+    environment:
+      - POSTGRES_USER=slick_tests
+      - POSTGRES_PASSWORD=slick_tests
+      - POSTGRES_DB=slick_tests
+    healthcheck:
```

Comment:
Shoot. I left myself a TODO and then missed this. The compose docs [have a page on dealing with this](https://docs.docker.com/compose/startup-order/) but if we used one of those wrapper scripts, I _think_ the most straightforward way to use it would be in the `scripts/` or to modify the container entrypoints. But, there's likely some case where you don't want it and its extra code to manage. 

So, I think I'll just revert to compose 2.3 for now. They haven't dropped support for an old version of the compose file format since 1.0 yet.